common/app/views/fragments/headTonal.scala.html

```diff
@@ -1,7 +1,8 @@
-@(item: model.ContentType, showBadge: Boolean = false)(implicit request: RequestHeader)
+@(item: model.ContentType, showBadge: Boolean = true)(implicit request: RequestHeader)
```

Comment:
Beware the potential side effects of changing a parameter's default value. This fragment is called in four different places as far as I could see.